lib/styles/editor/_editor-content.scss

```diff
@@ -64,14 +46,12 @@ $list-item-color: #1a141f;
   // Blockquote
   blockquote {
     font-weight: $neeto-ui-font-medium;
-    font-style: italic;
```

Comment:
We need italics for the blockquote item right?

Previously
<img width="883" alt="Screenshot 2021-12-16 at 12 40 20 PM" src="https://user-images.githubusercontent.com/33386004/146324568-cb61d902-6271-443b-b3a8-5ddbceac11f0.png">

Reply:
It was removed purposely. Can be reverted if needed.